traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -909,8 +945,13 @@ sub capacity {
 	my $id = $self->param('id');
 
 	if ( $self->is_valid_delivery_service($id) ) {
-		if ( $self->is_delivery_service_assigned($id) || &is_admin($self) || &is_oper($self) ) {
+		my $tenant_utils = Utils::Tenant->new($self);
+		my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+		if ( $self->is_delivery_service_assigned($id) || $tenant_utils->ignore_ds_users_table() || &is_admin($self) || &is_oper($self) ) {
```

Comment:
I'll remove "is_admin", I didn't do so in the first place because I was trying to minimize the risk on existing functionality, so I did not change existing code.

I do not understand the remark about the "if $tenant_utils->ignore_ds_users_table() is true then it doesn't matter if is_delivery_service_assigned($id), does it" - you are correct, but this is the reason things are ORed